config/settings.py

```diff
@@ -257,6 +257,8 @@
 # UA-XXXXXXXXX-X
 GOOGLE_TRACKING_ID = os.environ.get("GOOGLE_TRACKING_ID")
```

Comment:
i think google is not used anymore- we should remove that

Reply:
i use it, but yes understood, most people don't use google analytics. we can remove.    I just use it to aid in changes and troubleshooting.    
![image](https://github.com/user-attachments/assets/98198861-e677-47dc-961f-8f5af58ebd4b)